src/main/resources/gaming_games.avro

```diff
@@ -0,0 +1,51 @@
+{
+    "namespace": "gaming",
+    "name": "gaming_games",
+    "type": "record",
+    "fields": [
+        {
+            "name": "id",
+            "type": {
+                "type": "int",
+                "arg.properties": {
+                    "iteration": {
+                        "start": 1000,
+                        "step": 1
+                    }
+                }
+            }
+        },
```

Comment:
This should line up with `game_room_id`'s possible values below, to give us sensible joins. Otherwise there will be a bunch of games with no other data that correlates. I believe that the intent of this is to only emit a finite set of games e.g. to populate a table. since datagen connector / `avro-random-generator` don't really have that finite concept (afaik), I think this should just line up with below and repeat, i.e.:
```
        {
            "name": "id",
            "type": {
                "type": "int",
                "arg.properties": {
                    "range": {
                        "min": 1000,
                        "max": 5000
                    }
                }
            }
        },
```